pages/kubernetes/how-to/use-sfs-with-kubernetes.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: How to use SFS with Kubernetes Kapsule
+description: This page explains how to use the Scaleway File Storage Container Storage Interface (CSI) driver to enable Kubernetes users to manage Scaleway File Storage volumes within their clusters.
+tags: kubernetes kubernetes-kapsule kapsule sfs
+dates:
+  validation: 2025-07-18
+  posted: 2025-07-18
+categories:
+  - containers
+  - kubernetes
+---
+import Requirements from '@macros/iam/requirements.mdx'
+
+The Scaleway File Storage Container Storage Interface (CSI) driver enables Kubernetes users to manage Scaleway File Storage volumes within their clusters.  
+The Scaleway File Storage CSI driver is designed to work with Kubernetes Kapsule and Kosmos clusters, providing a standardized interface to create, manage, and attach file storage volumes to your containerized workloads. For more details on Scaleway File Storage, refer to the [Scaleway File Storage documentation](https://www.scaleway.com/en/file-storage/).
+
+## Supported features
+
+The Scaleway File Storage CSI driver supports the following features:
+
+- Dynamic provisioning: Automatically create Scaleway File Storage volumes using PVCs and StorageClasses.
+- `ReadWriteMany` access mode: Allows multiple pods to read from and write to the same file storage volume simultaneously.
```

Comment:
maybe we can clarify that volumes can be mounted on multiple nodes too?

> ReadWriteOnce – the volume can be mounted as read-write by a single node
> ReadOnlyMany – the volume can be mounted read-only by many nodes
> ReadWriteMany – the volume can be mounted as read-write by many nodes

https://kubernetes.io/blog/2021/09/13/read-write-once-pod-access-mode-alpha/